src/prefect/engine/cloud/task_runner.py

```diff
@@ -262,6 +274,13 @@ def run(
                 map_index=context.get("map_index"),
             )
             context.update(task_run_version=task_run_info.version)  # type: ignore
+
+            start_state = (
```

Comment:
I'm not clear on what this variable is for

Reply:
I suspect this still works even without this line because the Metastate is unpacked automatically by the TaskRunner

Reply:
Good catch - this was leftover from some prior experimentation.